src/main/java/org/truffleruby/parser/RubyWarnings.java

```diff
@@ -147,12 +70,9 @@ public void warn(ID id, String fileName, int lineNumber, String message) {
         buffer.append("warning: ").append(message).append('\n');
         DebugHelpers.eval(runtime, "$stderr.write Truffle::Interop.from_java_string(message)", "message", buffer.toString());
     }
-
-    /**
-     * Prints a warning, unless $VERBOSE is nil.
-     */
```

Comment:
Please keep the javadoc and add it the `warn` above too.
It means we don't need to look at the source to know what it does.